google_image_nixos_custom/main.tf

```diff
@@ -25,7 +31,10 @@ variable "licenses" {
 # ----------------------------------------------------
 
 data "external" "nix_build" {
-  program = ["${path.module}/nixos-build.sh", var.nixos_config]
+  program = concat(
+    ["${path.module}/nixos-build.sh", var.nixos_config],
+    var.NIX_PATH == "" || var.NIX_PATH == "-" ? [] : ["-I", var.NIX_PATH]
```

Comment:
It needs to be set as an environment variable. `-I` only applies a single key=value pair.

Reply:
done in 79a8ad70f8fe472607d70ddc2003172261e86e9e